Updated some lines (#90004)
Azure/azure-content: articles/virtual-machines/windows/prepare-for-upload-vhd-image.md

   ```powershell
   Enable-PSRemoting -Force
   ```
   
> [!NOTE]

> > By running this command; it will enable the appropiate firewall rules automatically

Reviewer: ```suggestion
> When this command runs, it enables the appropriate firewall rules automatically.
```
Fixing typo, grammar, punctuation, Azure style

   New-NetFirewallRule -DisplayName AzurePlatform -Direction Inbound -RemoteAddress 168.63.129.16 -Profile Any -Action Allow -EdgeTraversalPolicy Allow
   New-NetFirewallRule -DisplayName AzurePlatform -Direction Outbound -RemoteAddress 168.63.129.16 -Profile Any -Action Allow
   ```
  
 > [!IMPORTANT]

>  > 168.63.129.16 is a special public IP address that is owned by Microsoft for Azure Platform
>  > ([what-is-ip-address-168-63-129-16](../../virtual-network/what-is-ip-address-168-63-129-16.md))

Reviewer: ```suggestion
 > 168.63.129.16 is a special public IP address that is owned by Microsoft for Azure. For more information, see [What is IP address 168.63.129.16](../../virtual-network/what-is-ip-address-168-63-129-16.md).
```
Fixing terminology, formatting, anchor text for SEO
